Makefile

```diff
@@ -336,8 +335,8 @@ out/doc/api/%.html: doc/api/%.md
 		fi
 	[ -x $(NODE) ] && $(NODE) $(gen-html) || node $(gen-html)
 
-docopen: out/doc/api/all.html
-	-google-chrome out/doc/api/all.html
+docopen: $(apidocs_html)
```

Comment:
Maybe we can sidestep this issue by makeing `doc` do what `doc-only` does now. The fact that `doc` does a full build seems unexpected. Can be done in another PR I assume.